README.md

```diff
@@ -110,17 +110,17 @@ We're mostly a Rails app, with a bit of Webpack sprinkled in. **For most cases,
 
 Here are some singleton commands you may need, usually in a separate instance/tab of your shell.
 
-- Running the job server (if using `bin/rails server`) -- this is mostly for notifications and emails: **`bin/rails jobs:work`**
-- Clearing jobs (in case you don't want to wait for the backlog of jobs): **`bin/rails jobs:clear`**
+- Running the job server (if using `bin/rails server`) -- this is mostly for notifications and emails: **`bin/rails jobs: work`**
+- Clearing jobs (in case you don't want to wait for the backlog of jobs): **`bin/rails jobs: clear`**
 
-Current gotchas: potential environment issues with external services need to be worked out.
+Current gotchas: potential environmental issues with external services need to be worked out.
```

Comment:
This line should remain `environment`, environmental is something that Grammarly will suggest, but in this case, it's incorrect.